src/main/java/jdbchomework/Main.java

```diff
@@ -20,4 +20,8 @@ public static void main(String[] args) throws IOException {
         reader.close();
         HibernateUtil.closeSessionFactory();
     }
+
+    public static int some() {
+        return 1;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3400)